### ISSUE TITLE
patch included from README.md won't work, one provided by statuscmd does

### ISSUE THREAD

**Issue by wael444** [torrinfail/dwmblocks#60]
patching dwm with [statuscmd](https://dwm.suckless.org/patches/statuscmd/) and also using the dwmblocks patch provided in the README.md [here](https://gist.github.com/IGeraGera/e4a5583b91b3eec2e81fdceb44dea717) doesn't work and i can't tell why.
however using the one in the [Status monitor patches](https://dwm.suckless.org/patches/statuscmd/) section works? should i open a pull request to change it or is it just me?

**Comment by m1dsolo** [comment]
> patching dwm with [statuscmd](https://dwm.suckless.org/patches/statuscmd/) and also using the dwmblocks patch provided in the README.md [here](https://gist.github.com/IGeraGera/e4a5583b91b3eec2e81fdceb44dea717) doesn't work and i can't tell why. however using the one in the [Status monitor patches](https://dwm.suckless.org/patches/statuscmd/) section works? should i open a pull request to change it or is it just me?

You are right. I use [20241009 statuscmd dwm patch](https://dwm.suckless.org/patches/statuscmd/dwm-statuscmd-20241009-8933ebc.diff) and [20210402 statuscmd dwmblocks patch](https://dwm.suckless.org/patches/statuscmd/dwmblocks-statuscmd-20210402-96cbb45.diff), it works!
